coop_badge_reader/security/ir_module_category.xml

```diff
@@ -2,6 +2,7 @@
 <!--
     Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
     @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+    Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
```

Comment:
No need to add copyrights on file we don't / barely touch
Anyways it's ok here, no need to change it